docs/changehistory/NextVersion.md

```diff
@@ -1494,7 +1494,7 @@ Now, you can bind a set of Ids as a parameter for the `IN` clause. The Ids will
 The signature of the method has changed to:
 
 ```ts
-query(ecsql: string, params?: QueryBinder, rowFormat = QueryRowFormat.UseArrayIndexes, config?: QueryOptions): AsyncIterableIterator<any>;
+query(ecsql: string, params?: QueryBinder, rowFormat = QueryRowFormat.UseECSqlPropertyIndexes, config?: QueryOptions): AsyncIterableIterator<any>;
```

Comment:
Is the part `rowFormat = QueryRowFormat.UseECSqlPropertyIndexes` correct? It looks like `rowFormat` was moved into `QueryOptions`.

Reply:
Fixed.